.github/workflows/remote-build-ios-hybrid.yml

```diff
@@ -0,0 +1,32 @@
+name: Remote Build iOS Hybrid
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [main]
+
+jobs:
+  build:
+    runs-on: macos-15-xlarge
+    steps:
+      - name: Checkout
+        # v4
+        uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608
+        with:
+          submodules: true
+          ref: ${{ github.event.pull_request.head.sha }}
```

Comment:
I'd appreciate a confirmation that this is the right ref 🤞🏻

Reply:
nope 😄 

commented above, but just drop the ref and it will behave as you expect